### PR TITLE
#30, #29, #19 filter for sublease, seniors, swap apartments

### PR DESCRIPTION
**Pull request by valoricDe**

### CODE REVIEW

**Comment by neopostmodern** [comment]
I wonder if "only for elderly" and "only swaps" are really necessary / make sense? Isn't the usual case that you just need some sort of apartment? E.g. I'd _assume_ that someone looking for an apartment to swap would also be okay with getting a non-swap one and then just letting go of the current one. Put differently, they're bonuses.
In that sense I think it'd be good to reduce the number of configuration options?

For subleases the story might be different, but I'm not sure if the Wohnungsbot is really suitable for that kind of search yet anyway. So I guess even there I'd cut the option, but I don't have a strong opinion on it.

**Comment by valoricDe** [comment]
I aggree. I removed the only swap and senior filters.
I would like to implement something additional. I know that you can set a cookie to hide the consent banner. But I do not know how to integrate the cookie setting into the bot?

**Comment by valoricDe** [comment]
And it would be cool to add a button to skip certain listings. We could show the in progress listing on the right panel with a pause and skip button or similar

File: app/components/configurationStages/review.js [hunk 1]
```diff
@@ -127,6 +127,24 @@ const reviewStage: StageDescription = {
...
+              <br />
+            </>
+          ) : null}{' '}
+          {configuration.filter.notSpecificallyForSeniors? (
+            <>
+              keine Seniorenwohnung
+              <br />
+            </>
+          ) : null}{' '}
+          {configuration.filter.noSublease? (
```
Comment (neopostmodern) [inline app/components/configurationStages/review.js:142]: this is missing the onlySublease option, isn't it?

File: app/middleware/electron.js [hunk 2]
```diff
@@ -44,7 +44,9 @@ export default (store: Store) => (next: (action: Action) => void) => async (
       // eslint-disable-next-line no-console
       console.error(`No view registered for ${name}!`);
     } else {
-      viewState.browserView.webContents.loadURL(targetUrl);
+      viewState.browserView.webContents.loadURL(targetUrl).then(() => {
```
Comment (valoricDe) [inline app/middleware/electron.js:47]: That doesn't work as I would have expected. But somewhere this cookie should be set

Comment (neopostmodern) [reply app/middleware/electron.js:47]: could you move this into a separate PR please?

**Comment by neopostmodern** [comment]
Concerning your other two ideas:
I've opened a new issue for the cookie-consent: #43 
The real-time interaction to skip applications is discussed here: #18 — but it's a tough one, I'll elaborate there.

**Pull request merged**